changebot/changelog.py

```diff
@@ -25,8 +25,12 @@ def find_prs_in_changelog_by_section(content):
     subcontent = ''
     previous = None
 
+    new_changelog_format = False
+    
     for line in content.splitlines():
-        if '-------' in line:
+        if '=======' in line:
+            new_changelog_format = True
+        if '=======' in line or (not new_changelog_format and '-------' in line):
```

Comment:
But then the expression will always be True, not just for the lines that has "====" in them. Ideally the first if need to run only until the first occurrence of "====", from there on we know it's a new changelog.